src/Components/OATPropertyEditor/Internal/EditorJsonTab/EditorJsonTab.styles.ts

```diff
@@ -0,0 +1,25 @@
+import {
+    IEditorJsonTabStyleProps,
+    IEditorJsonTabStyles
+} from './EditorJsonTab.types';
+import { CardboardClassNamePrefix } from '../../../../Models/Constants/Constants';
+
+const classPrefix = `${CardboardClassNamePrefix}-editorjsontab`;
+const classNames = {
+    root: `${classPrefix}-root`
+};
+
+// export const EDITORJSONTAB_CLASS_NAMES = classNames;
```

Comment:
I actually added this to the template so that if you need to reference the names across components for some reason, each would be exported as a unique name. TOday we export them as `classNames` which makes the import very hard to find the right one so I thought this was better. Do you think it's worth leaving commented out in case we need it? or just add it back at that point?

Reply:
Add it back, I like the pattern of not exporting `classNames` and this looks like a clean solution.